en/editor/project/joint-texture-layout.md

```diff
@@ -0,0 +1,71 @@
+# Joint Texture Layout Settings
+
+To ensure that the [Skeletal Animation](../../engine/animation/skeletal-animation.md#About-Dynamic-Instancing) can also participate fully and correctly in [Dynamic Instancing](../../engine/renderable/model-component.md#Instancing-Batching), requiring the user to manually specify how the data for each joint texture is assigned.
+
+For example, a scene with a large number of identical characters to draw, each of which may be walking, jumping, or attacking. If you want to be able to draw all characters correctly with one drawcall, an important prerequisite is that the data for all three animations is stored in the same joint texture. Currently, in the default [pre-baked skeletal animation mode](../../engine/animation/skeletal-animation.md#pre-baked-skeletal-animation-system), the joint textures are automatically reused globally, but the size of each texture and which animations they store are unpredictable. If instancing of the skinned model is turned on directly without any processing, the final runtime effect is that some animations are correct and some are completely wrong and unpredictable. Therefore, we introduce the Joint Texture Layout Settings panel, which is used to manually specify which animation information is stored for which skeletons in each joint texture.
+
+> Note: The Joint Texture Layout Settings panel provides what is essentially a runtime "memory allocation guideline". For the specified skeletons and animation resources, they are guaranteed to be allocated according to the specified rules, but if a resource is used at runtime that is not specified by the rules, it will go back to the automatic allocation mode of global reuse.
+
+Let's take a [specific scene](https://github.com/cocos-creator/example-3d/blob/master/demo02/assets/scenes/instanced-skinning.scene) as an example to show the setup process and how it works in practice .
+
+The figure below shows a sample scenario with multiple instances from the same model, each playing a completely different animation at the same time. These models use the real-time computation animation mode and do not have instancing turned on. As you can see, the current scene plus the UI has a total drawcall of 60 and an instance count of 0. This state will be used as a baseline for later changes as a comparison.
+
+[Baseline](./index/instancing_baseline.gif)
+
+To create a model with the instancing turned on, you need to:
+* Make sure `useBakedAnimation` is checked on the SkeletalAnimationComponent.
+* Check `USE_INSTANCING` for all materials used by SkinningModel.
+
+![Enabling](./index/enabling_instancing.png)
+
+In our example scene we actually made two sets of prefab and set the material diffuse color to blue in the instancing version in order to see and distinguish between the two systems at the same time. You can see that the effect has been exactly correct, and that only 5 drawcalls (each model is divided into 5 parts) were used, with 45 instances.
```

Comment:
In our example scene we actually made two sets of prefab and set the material diffuse color to blue in the instancing version in order to see and distinguish between the two systems at the same time. You can see that the effect has been exactly correct, and that only 5 drawcalls (each model is divided into 5 parts) were used, with 45 instances.
-->
In the example scene (referenced above) two sets of prefabs are made and the material diffuse color set to blue in the instancing version in order to see and distinguish between the two systems at the same time. Notice that the effect has been exactly correct and that only 5 drawcalls (each model is divided into 5 parts) were used, with 45 instances.